### PR TITLE
fix rotation bug after translation

### PR DESCRIPTION
**Pull request by AnthonyGlt**
## Before contributing

Read [CONTRIBUTING.md](https://github.com/iTowns/itowns/blob/master/CONTRIBUTING.md) and [CODING.md](https://github.com/iTowns/itowns/blob/master/CONTRIBUTING.md) to apply `iTowns` conventions on PRs, Git history and code.

### CODE REVIEW

File: examples/js/XR/Controllers.js [hunk 1]
```diff
@@ -332,15 +332,18 @@ function switchRegisteredCoordinates() {
...
 
 // ////////////////////////////////// MODE 1
 
 function getRotationYaw(axisValue) {
-    if(axisValue === 0) {
+    if (axisValue === 0) {
         return;
     }
-    deltaRotation += Math.PI / (160 * axisValue);
+    if (axisValue) {
```
Comment (jogarnier) [inline examples/js/XR/Controllers.js:343]: is this test revelant with the previous one l.340 ?

File: examples/js/XR/Controllers.js [hunk 2]
```diff
@@ -372,20 +375,20 @@ function cameraOnFly(ctrl) {
         return;
     }
     let directionX = new itowns.THREE.Vector3();
-    let directionY = new itowns.THREE.Vector3();
+    let directionZ = new itowns.THREE.Vector3();
```
Comment (jogarnier) [inline examples/js/XR/Controllers.js:378]: I meant direction relative to the stick + the 3d view as the user expect it,
not direction programmically speaking where Z is pointing toward  :) I felt it was less confusing

Comment (AnthonyGlt) [reply examples/js/XR/Controllers.js:378]: Yes but it's a translation on the z axis. We should stay consistent with the scene

File: examples/js/XR/Controllers.js (hunk 1 above)
Comment (AnthonyGlt) [reply examples/js/XR/Controllers.js:343]: traiter le cas 0 et undefined sans faire de return

File: examples/js/XR/Controllers.js (hunk 2 above)
Comment (jogarnier) [reply examples/js/XR/Controllers.js:378]: ok fine

**Review by jogarnier — APPROVED** [review]

**Pull request merged**